packages/runtime/src/widgets/Search.tsx

```diff
@@ -54,7 +54,7 @@ export const Search: React.FC<SearchProps> = ({
     name: itemTemplate?.name,
   });
   const { rootRef, values } = useRegisterBindings(
-    { styles, value, options },
+    { styles, value, options, placeholder },
```

Comment:
Let's destructure the props for the props that shouldn't be expressions but then otherwise pass everything in as bindings, so be inclusive instead of exclusive. There's a few examples of this like `Column`. This avoids us having to update it further for every new property.